src/views/Short_generator/__test__/__snapshots__/ShortGenerator.test.js.snap

```diff
@@ -1,3 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ShortGenerator render button 1`] = `"<div class=\\"generatorContainer\\"><div class=\\"topZone\\"><site-input-stub></site-input-stub> <div class=\\"options\\"><option-button-stub name=\\"free\\" route=\\"\\" customwidth=\\"100px\\" customheight=\\"70px\\"></option-button-stub> <option-button-stub name=\\"custom\\" route=\\"\\" customwidth=\\"100px\\" customheight=\\"70px\\"></option-button-stub> <option-button-stub name=\\"short\\" route=\\"\\" customwidth=\\"100px\\" customheight=\\"70px\\"></option-button-stub></div></div> <div class=\\"itinerary\\"><idea-item-stub></idea-item-stub> <idea-item-stub></idea-item-stub> <idea-item-stub></idea-item-stub> <idea-item-stub></idea-item-stub> <idea-item-stub></idea-item-stub> <idea-item-stub></idea-item-stub> <idea-item-stub></idea-item-stub> <idea-item-stub></idea-item-stub></div> <time-line-stub></time-line-stub></div>"`;
+exports[`ShortGenerator render component 1`] = `
```

Comment:
I think this is quite brilliant... 🔥

Reply:
that last comma don do anything. Is a good practice to add it in order to don 't forget them between lines 
by Eric Dürr